src/server/api/controllers/profile.controller.js

```diff
@@ -61,6 +61,30 @@ const getAdminsProfile = async (userId) => {
   }
 };
 
+const postAdminsProfile = async (adminRole, userId) => {
```

Comment:
Can I suggest a different name for the first parameter: `body`, `values`, `data`, `profile` or `updatedProfile` since `adminRole` is just a property name on this object.